src/all/hitomi/src/eu/kanade/tachiyomi/extension/all/hitomi/Hitomi.kt

```diff
@@ -598,9 +615,35 @@ class Hitomi(
 
     override fun popularMangaParse(response: Response) = throw UnsupportedOperationException()
     override fun popularMangaRequest(page: Int) = throw UnsupportedOperationException()
+
+    override fun setupPreferenceScreen(screen: PreferenceScreen) {
+        ListPreference(screen.context).apply {
```

Comment:
change to switch preference with different summaries
[example](https://github.com/keiyoushi/extensions-source/blob/b698c31d5359718e62c44098a47e7bed1f27572a/src/all/ninenineninehentai/src/eu/kanade/tachiyomi/extension/all/ninenineninehentai/NineNineNineHentai.kt#L282-L288)

Reply:
Done and test passed. thanks for your patience.